blog/2023-04-13-rse-on-mastodon: state of RSE on mastodon (#412)
nordic-rse/nordic-rse.github.io: content/blog/2023-04-13-rse-on-mastodon.md

+++
title = "State of Research software engineers on Mastodon"

[extra]

author = "Richard Darst"
+++

Many domains have migrated to Mastodon as of 2023.  Have Research
Software Engineers?  This post examines the situation, based on [this
post by rkdarst](https://fosstodon.org/@rkdarst/110146601969193973)

Summary: not much yet

This post can be updated as we get more information/the situation
evolves, please send pull requests.

## RSE organizations

* US RSE: <https://fosstodon.org/@us_rse>

> * DE RSE: <https://fosstodon.org/@de_rse@mastodon.social>

Reviewer: ```suggestion
* DE RSE: <https://mastodon.social/@de_rse>
```

* Nordic RSE: <https://fosstodon.org/@nordic_rse>
* Society of RSE (UK/int): ??

> * ... ?

Reviewer 2: ```suggestion
* Research Software Alliance: <https://fosstodon.org/@researchsoft>
* Netherlands eScience Center: <https://fosstodon.org/@eScienceCenter@akademienl.social>
* ... ?
```

Reviewer: The Netherlands one should be
`https://akademienl.social/@eScienceCenter`

## Other accounts to follow to seed your community

> TODO (should we join some other list instead of make a new one here?)

Reviewer 3: I think getting a category at <https://fedi.directory/> as you mention below is the best option for this. It will be inherently more visible than any bespoke list we might start, and it avoids any decisions about where such a list would be hosted and maintained.  (This is @mattasdata from Mastodon btw. Need to reconcile my different usernames :stuck_out_tongue: )

Reviewer 2: Does this mean removing this section for now, or keeping the TODO?

> TODO (should we join some other list instead of make a new one here?)

Author: ```suggestion
TODO: We should try to get a category in <https://fedi.directory/> rather than make our own list.
```

## Common hashtags

> * **#RSEng and #RSE** have been seen, but #RSEng is preferred

Reviewer 3: Perhaps worth a mention that `#RSEng` was the primary hastag on Twitter for RSE-related stuff

Reviewer: Could also say that when both are used, #RSE is the people and #RSEng is the activity, though #RSE is often used for both.

Reviewer 2: Cannot really comment on this, but it's probably better to publish this rather than wait. Keep as it is?

> * **#RSEng and #RSE** have been seen, but #RSEng is preferred

Reviewer 4: ```suggestion
* **#RSEng** and **#RSE** have been seen, but #RSEng is preferred
```

  ([ref](https://fosstodon.org/@HeathrTurnr/110147262319789502) (same

>   as Twitter).  There doesn't seem to be much activity there.

Author: ```suggestion
  as Twitter).  There doesn't seem to be much activity there.  #RSEng was
  the primary tag from from Twitter.  Sometimes #RSE is the people and
  #RSEng is the activity.
```

Reviewer 4: “(same as Twitter)” not needed given next sentence explains further.

* **#HPC** has a fair community.

* **#OpenScience** can be found, in addition to **ResearchData** and
  **ResearchSoftware**.

> * We propose **#SciComp** for as a general "scientific computing" tag

Reviewer: I don't think it's so useful to have a term for scientific computing that is highlighted. It's the research computing that matters here, whether it's science, engineering, humanities, etc.

  which can partly overlap with the above, but is more general to
  people interested in the computing part of science.

* There is the [a.gup.pe](https://a.gup.pe/u/researchsoftware) group
  "researchsoftware", which will boost any post tagging it, and can be
  used as a more targeted distribution group than tags
  ([ref](https://fosstodon.org/@mattasdata@qoto.org/110151591358979693)).
  We could also get a listing at <https://fedi.directory/>.

## RSE-focused Mastodon servers

There don't seem to be any servers specifically focused on the RSE
community.  Perhaps the closest one is the HPC community
<https://mast.hpc.social>, and we are invited to take part there

> ([ref](https://mast.hpc.social/@AlanSill/110147881007346377)).  Many

Author: ```suggestion
([ref](https://mast.hpc.social/@AlanSill/110147881007346377)).  There is overlap
between the HPC and RSE communities, and the initial community survey (ref)
for creation of the HPC.social Mastodon server showed a significant interest in
exchanging information on software topics.

Many
```

> ([ref](https://mast.hpc.social/@AlanSill/110147881007346377)).  Many

Reviewer 5: There is overlap between the HPC and RSE communities, and the initial community survey ([ref](https://survey.hpc.social/results/hpc-social-design-survey.html)) for creation of the HPC.social Mastodon server showed a significant interest in exchanging information on software.

of the repliers used <https://fosstodon.org>.

> One could ask if if's better to have a common server, or be more

Reviewer 3: ```suggestion
One could ask if it's better to have a common server, or be more
```

distributed among the communities of other scientists to spread the
word.  There is a [list of academic/science related servers at

> fedi-science](https://fediscience.org/server-list.html).

Author: ```suggestion
fedi-science](https://fediscience.org/server-list.html). The primary advantage
of a topic-driven server is a better local timeline, which will be populated with
the posts of everyone on that server.  While following hashtags/etc. is similar,
the local server makes this more natural.
```

> 

Reviewer 5: The primary advantage of having a server focused on any topic, for example RSE, is the ability to browse the local timeline without needing to wait for federation of related servers to make relevant posts local. Of course, any server can be used as a home base in the Fediverse, and any account can follow posts using a given hasttag by simply following that hashtag, so having a server for a community is largely a convenience, and doing so provides a natural gathering mechanism for people interested in a given topic.

Author: (added to text)

There could be room for some RSE organization to set up a
RSE-focused server. (editor note: I, rkdarst, would somewhat prefer a
"scicomp"-focused mastodon server to bring together data, computing,

> software, etc. people.)

Author: ```suggestion
software, etc. people.  Yet, this is also too broad and nebulous and without
organizational backing so comments were negative on this idea.)
```

> software, etc. people.)

Reviewer 3: In my opinion, there is a better chance of an RSE-focused server being set up because there are actual RSE organisations. "Scientific computing" is a much more nebulous and dispersed domain, and I suspect the only people who would actually sign up to it would be RSEs anyway (hence why not just make an RSE instance).

Some next actions would be making a formal proposal to some or all of the RSE organisations. There is the international RSE council which would be a catch-all: https://researchsoftware.org/council.html

Reviewer 4: Also "scientific computing" would not be a good fit for RSEs who work in social science, humanities, multidisciplinary projects, etc.

When thinking of running an instance you need to think about governance, moderation, sustainability, etc - attaching to existing an community or (group of) organisation(s) should help with this.

Author: I agree with the comments that this is too broad and we couldn't get someone to run it.  Commented to say this.

Reviewer: The HPC community has https://hpc.social which includes https://mast.hpc.social/about, supported by organizational contributions,

## Future

* Please send pull request with updates.
* We should contact SocRSE UK and ask them if they are interested in

>   a Mastodon community.

Reviewer 4: There was some discussion on Slack at the end of 2022. More people have moved to Mastodon since then - it would be good to have more institutions/organizations engaging on Mastodon, but I'm not sure if anyone's ready to run an instance.

Reviewer 2: There are some interesting points in this PR, but I cannot convert them to changes in the blog post. This section would probably be the best place for them in this blog post. Unfortunately they will be effectively lost after merging the PR.

Author: I added them as changes.  I think the comments would be preserved, I will

+++
title = "State of Research Software Engineers on Mastodon"

[extra]

author = "Richard Darst"
+++

Many domains have migrated to Mastodon as of 2023.  Have Research
Software Engineers?  This post examines the situation, based on [this
post by rkdarst](https://fosstodon.org/@rkdarst/110146601969193973)

Summary: not much yet

This post can be updated as we get more information/the situation
evolves, please send pull requests.

## RSE organizations

* US RSE: <https://fosstodon.org/@us_rse>
* DE RSE: <https://mastodon.social/@de_rse>
* Nordic RSE: <https://fosstodon.org/@nordic_rse>
* Society of RSE (UK/int): ??
* Research Software Alliance: <https://fosstodon.org/@researchsoft>

> * Netherlands eScience Center: <https://akademienl.social/@eScienceCenter>

Author: ```suggestion
* Netherlands eScience Center: <https://akademienl.social/@eScienceCenter>
* RSE Midlands <https://mastodon.social/@RSE_Midlands>
```

* ... ?

## Other accounts to follow to seed your community

TODO (should we join some other list instead of make a new one here?)

## Common hashtags

* **#RSEng** and **#RSE** have been seen, but #RSEng is preferred

>   ([ref](https://fosstodon.org/@HeathrTurnr/110147262319789502) (same

Author: ```suggestion
  ([ref](https://fosstodon.org/@HeathrTurnr/110147262319789502).
```
